src/styles/blocks/_review.scss

```diff
@@ -0,0 +1,35 @@
+.review {
+  margin-top: 16px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  &__stars{
+    display: flex;
+    justify-content: space-between;
+    width: 96px;
+   &-star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+    &:nth-of-type(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
+    &:nth-last-child(1) {
+      margin-right: 0;
+    }
+   }
+  }
```

Comment:
use code from previous task, just ctrl+c ctrl+v from stars, no need to modify it